srcpkgs/kernel-libc-headers/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'kernel-libc-headers'
 pkgname=kernel-libc-headers
 version=4.19.0
-revision=2
+revision=3
 bootstrap=yes
-nostrip=yes
```

Comment:
leave this change out. somebody likely wants to keep the `git blame` history clean. only do short_desc or none at all.